docs/source/howto/installation.rst

```diff
@@ -236,7 +236,7 @@ However, optimal performance at that scale might require some tweaks to the AiiD
 
 Here are a few general tips that might improve the AiiDA performance:
 
-  1. :ref:`Move the Postgresql database<how-to:installation:more:move_postgresql>` to a fast disk (SSD), ideally on a large partition.
+  1. Move the Postgresql database to a fast disk (SSD), ideally on a large partition.
```

Comment:
I guess this was his intention for that file he just forgot to include it in the tree. Is there a place we can put this advanced content? Should we just inline it below the itemization?

Reply:
I guess we can inline it for now.